src/App/Fossa/Analyze.hs

```diff
@@ -269,11 +268,16 @@ analyze cfg = Diag.context "fossa-analyze" $ do
       skipResolutionSet = Config.vsiSkipSet $ Config.vsiOptions cfg
       vendoredDepsOptions = Config.vendoredDeps cfg
       grepOptions = Config.grepOptions cfg
+      BaseDir fossaDepsDir = Config.fossaDepsDir cfg
 
   -- additional source units are built outside the standard strategy flow, because they either
   -- require additional information (eg API credentials), or they return additional information (eg user deps).
-  vsiResults <- Diag.errorBoundaryIO . diagToDebug $
-    Diag.context "analyze-vsi" . runStickyLogger SevInfo . runFinally $ do
+  vsiResults <- Diag.errorBoundaryIO
+    . diagToDebug
+    $ Diag.context "analyze-vsi"
+      . runStickyLogger SevInfo
+      . runFinally
+    $ do
```

Comment:
Don't know why these are re-formatting when I am using `make fmt`. Auto format occurs at different points in this pr as well fyi